1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,32 @@
 function readingList(books) {
-  // Write your code here...
+  let ulElement = document.createElement("ul")
+  content.appendChild(ulElement)
```

Comment:
What is the `content` here? Do you think  document.querySelector  for content is necessary?